src/Chapter04.Tests/Listing04.24.Tests.cs

```diff
@@ -0,0 +1,29 @@
+using IntelliTect.TestTools.Console;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter04.Listing04_24.Tests
+{
+    [TestClass]
+    public class Listing04_23_Tests
```

Comment:
First, class name doesn't match the file. Second, 4.23 was updated to be `ProgramTests` we should be consistent in class naming.